R/logging.R

```diff
@@ -0,0 +1,110 @@
+#' Get node log files from Azure Storage. By default, this operation will print the files on screen.
```

Comment:
From storage? Will they always be there, even if the task fails somehow?

Ok.. read the API below - looks like you're actually pulling from the node.

Reply:
This pulls from the node, it's in the docs

Reply:
But the doc string here says storage... is that incorrect?

Reply:
Fixed